opensrp-chw/src/kk/assets/rule/add_anc_registration_calculation.yml

```diff
@@ -4,15 +4,15 @@ description: surname calculation
 priority: 1
 condition: "step1_member_consent.contains('member_consent_yes')"
 actions:
-  - "calculation = step1_same_as_fam_name.contains('same_as_fam_name')? step1_fam_name : step1_surname"
+  - "calculation = step1_same_as_fam_name.contains('same_as_fam_name')? step1_fam_name : step1_surname : ''"
```

Comment:
Just wanted to make sure that this ternary operator has two colons, what is the expected here?

Reply:
I missed that one, let me correct it and push the changes